src/rules/prefer-absolute-version-dependencies.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const areVersRangesValid = require('./../validators/dependency-audit').areVersRangesValid;
+const LintIssue = require('./../LintIssue');
+const lintId = 'prefer-absolute-version-dependencies';
+const nodeName = 'dependencies';
+const message = 'You are using an invalid version range. Please use absolute versions.';
+const ruleType = 'dependencies-version-range';
+
+const lint = function(packageJsonData, lintType) {
+  const rangeSpecifier = '=';
```

Comment:
Yeah, I hacked around for about an hour to also get absolute ranges that don't begin with an equals sign, but couldn't get to an elegant solution. Two ideas I came up with:

1. Check if it is _not_ starting with any "special" characters (e.g. '~', '^', '>', etc.), but then I wasn't sure if I can reliable enumerate all special characters, and whether this would be sufficiently future proof.
2. RegExp, obviously, but we all know this introduces more problems than solutions ...

I also looked into the "semver" package and hoped for an "isAbsolute" method, but they are more about comparing version numbers and ranges. Weirdly enough e.g. "=1.0.10" is _not_ a valid version number according to semver, looks like the "=" gets stripped of before the version is passed on to semver.